taskfiles/scripts/.helmfile_cleanup_cache.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -eu
+
+# File to store the last execution timestamp
+TIMESTAMP_FILE="$HOME/.last_helmfile_cleanup"
```

Comment:
I suggest we create this file somewhere else, maybe in one of:
1) the Launchpad dir, and that we add it to .gitignore
2) the path-prefix given by `${XDG_CACHE_HOME}/some_path`, like `${XDG_CACHE_HOME}/launchpad/`
3) the path helmfile is configured to use that is by default `${XDG_CACHE_HOME}/helmfile/`, but can be obtained with `helmfile cache info` <- this one can maybe lead to an issue, which is when doing `helmfile cache cleanup` helmfile deleting it (not sure if that would happen)

Reply:
used the current Launchpad dir and updated .gitignore as `XDG_CACHE_HOME` is not used by default in macOS